vagrant/Vagrantfile

```diff
@@ -72,13 +72,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
             end		
           end		
         end
+        if ENV['PYTHON_REPOS']
+          config.vm.provision "file", source: "./rackhd-pm2-config.yml", destination: "~/rackhd-pm2-config.yml"
```

Comment:
like it !
the original rackhd-pm2-config.yml was sit in the vagrant box 0.16.
which here under src control is better .